### ISSUE TITLE
Chromium crashes after boot.

### ISSUE THREAD

**Issue by Timmoz** [jayatvars/balena-chromium-kiosk#1]
Hello, 

I get an error after booting Chromium: 

Aw, Snap!
Something went wrong while displaying this webpage.

I test it on a RPI 3B, 3B+ and 3A+. All with this same error.
with URL_LAUNCHER_URL=chrome://version/

Here the last log entries from Balena:

27.03.19 09:19:59 (+0100)  chromium-kiosk  Fontconfig warning: "/etc/fonts/fonts.conf", line 100: unknown element "blank"
27.03.19 09:20:02 (+0100)  chromium-kiosk  [154:363:0327/082002.160108:ERROR:bus.cc(396)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
27.03.19 09:20:02 (+0100)  chromium-kiosk  [154:173:0327/082002.273769:ERROR:service_manager_context.cc(291)] Attempting to run unsupported native service: /usr/lib/chromium-browser/chrome_renderer.service
27.03.19 09:21:02 (+0100)  chromium-kiosk  [154:154:0327/082102.349530:ERROR:web_resource_service.cc(196)] Connection error with the json parser process.

Anyone any idea what this is all about and how to solve it?
Thanks!

**Comment by jacobrosenthal** [comment]
yeah I started getting this too. remove the --disable-gpu --disable-software-rasterizer --disable-dev-shm-usage 
https://github.com/jayatvars/balena-chromium-kiosk/blob/master/app/start.sh#L23

**Comment by Timmoz** [comment]
Thanks, that solves it! :)
